src/utils/index.ts

```diff
@@ -221,3 +212,47 @@ export const isEmpty = (value: any) => {
 
   return false;
 };
+
+export function isPastTime(timestamp: number): boolean {
+  const currentTime = Date.now();
+  return timestamp < currentTime;
+}
+
+export function getDateNDaysLater(daysToAdd: number): Date {
+  const millisecondsPerDay = 24 * 60 * 60 * 1000; // 24hours in milliseconds
+  const currentDate = new Date();
+  const futureDate = new Date(
+    currentDate.getTime() + daysToAdd * millisecondsPerDay
+  );
+
+  return futureDate;
+}
+
+/**
+ * Polyfill for localStorage
+ * localStorage wont work in some browsers, in incognito
+ * and no-cookie modes
+ * @returns { getItem: (key), setItem: (key, value) }
+ */
+interface Storage {
+  [key: string]: any;
+}
+export const localStorageHelper = () => {
+  const store: Storage = {};
+  return {
+    getItem: (key: string) => {
+      try {
+        return localStorage.getItem(key);
+      } catch (error) {
+        return store[key];
+      }
+    },
+    setItem: (key: string, value: string) => {
+      try {
+        localStorage.setItem(key, value);
+      } catch (error) {
+        store[key] = value;
+      }
```

Comment:
Minor but performance-wise, it would be better to not use try catch in this case.
```
const setItem = localStorage?.setItem;
if (setItem) { ... } else { ... }
```